### DISCUSSION TITLE
repackaging fixed Konsole for better of user experience

### DISCUSSION THREAD

**Discussion by HarshitSingh16** [paulmcauley/klassy#115] [Ideas]
respected sir , 

you had said that you know how to fix the infamous korner bug and you have already told the plasma devs how to fix it and according to you it would hardly take 5 minutes.
my suggestion is that , why not you download the source code of konsole make the changes and repackage it as a component of klassy itself just with a different name and alternatively you can send this code to kde devs (as a pull request).

this would not only be good for klassy as more and more people would want to use it but it would also be good for existing klassy users.

thankyou

**Comment by paulmcauley** [comment]
I said it would take _the Konsole developers_ 5 minutes to fix their blur code as they just need to add one QRegion parameter. I am not familiar with the Konsole code enough to know where the text area region, nor do I have unlimited time to want to get involved with their code. Please remind the Konsole developers to fix it.
